sormas-api/src/main/java/de/symeda/sormas/api/event/EventExportDto.java

```diff
@@ -128,6 +126,19 @@ public EventExportDto(
 		this.jurisdiction = new EventJurisdictionDto(reportingUserUid, surveillanceOfficerUuid, regionUuid, districtUuid, communityUuid);
 	}
 
+	private String buildSrcInstitutionalPartnerTypeString(
```

Comment:
Move this to a separate EventHelper class (in the api); DTos should ideally not contain such logic. You also don't need a StringBuilder, you can directly return the respective value in the if/else if body.